package.json

```diff
@@ -46,6 +46,7 @@
     "@types/jasmine": "2.5.41",
     "@types/node": "7.0.4",
     "awesome-typescript-loader": "3.0.0-beta.18",
+    "cash-cp": "^0.2.0",
```

Comment:
This should probably be hardcoded to a specific version like the other dev dependencies.